### DISCUSSION TITLE
[Doc Issue]  文档中vite-plugin-style-import需更新版本，否则会报错

### DISCUSSION THREAD

**Discussion by clordho** [youzan/vant#10149] [General]
**文档章节：**

- [gitee地址](https://vant-contrib.gitee.io/vant/v3/#/zh-CN/quickstart#yin-ru-zu-jian)
- [github地址](https://youzan.github.io/vant/v3/#/zh-CN/quickstart#yin-ru-zu-jian)

**已知错误：**

- 按文档中示例的`npm i vite-plugin-style-import@1.2.0 -D`安装1.2.0版本的依赖，配置按需引用后，将出现`[vite:style-import] __require.resolve is not a function`错误

**原因分析：**

- vite-plugin-style-import已确认修复此问题 https://github.com/vbenjs/vite-plugin-style-import/pull/62

**尝试解决：**

- 安装依赖时使用`1.4.1`以上版本

**Comment by chenjiahan** [comment]
升级新版本会有其他 bug，参见：https://github.com/vbenjs/vite-plugin-style-import/issues/52

等这个 bug 修复了再更新文档里的版本。
